src/sentry/api/endpoints/organization_integration_details.py

```diff
@@ -1,15 +1,23 @@
 from uuid import uuid4
 
+from rest_framework import serializers
+
 from sentry.api.bases.organization import OrganizationIntegrationsPermission
 from sentry.api.bases.organization_integrations import OrganizationIntegrationBaseEndpoint
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.integration import OrganizationIntegrationSerializer
-from sentry.models import AuditLogEntryEvent, ObjectStatus, OrganizationIntegration
+from sentry.features.helpers import requires_feature
+from sentry.models import AuditLogEntryEvent, Integration, ObjectStatus, OrganizationIntegration
 from sentry.shared_integrations.exceptions import IntegrationError
 from sentry.tasks.deletion import delete_organization_integration
 from sentry.utils.audit import create_audit_entry
 
 
+class IntegrationSerializer(serializers.Serializer):
+    name = serializers.CharField(required=False)
+    domain = serializers.URLField(required=False)
```

Comment:
@MeredithAnya for both fields should we add `allow_blank=True`?

Reply:
@NisanthanNanthakumar I don't think we want to allow people to set their `name` or `domain` to an empty string. I think adding `partial=True` will allow them to just not updated those fields if they don't want (like just update the `domain` or just update the `name`.